ci/travis/osx/before_install.sh

```diff
@@ -28,6 +28,7 @@ brew ln expat --force
 brew ln libxml2 --force
 brew ln gettext --force
 brew ln libffi --force
+brew remove gdal
```

Comment:
The `gdal` formula is not installed as part of `qgis-214` anymore, and should not be installed.
Or, is this because `gdal` (1.x) is installed by default on Travis CI? If so, its removal should come before all other installs and probably be like:

`brew remove gdal || true`

to ensure it does not throw an error on Travis _if_ it is not installed.